From 154d9242de865c3cca8e597be911d69a53077ab1 Mon Sep 17 00:00:00 2001
From: Chia-I Wu <olv@lunarg.com>
Date: Fri, 2 Apr 2010 10:27:57 +0800
Subject: [PATCH 02/11] st/vega: Do not depend on libm.

This fixes the "no rule to make target -lm" error on darwin, reported by
Jeremy Huddleston.
---
 src/gallium/state_trackers/vega/Makefile |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git Mesa-7.8/src/gallium/state_trackers/vega/Makefile Mesa-7.8/src/gallium/state_trackers/vega/Makefile
index 037d8dc..f6d1ea0 100644
--- Mesa-7.8/src/gallium/state_trackers/vega/Makefile
+++ Mesa-7.8/src/gallium/state_trackers/vega/Makefile
@@ -42,7 +42,8 @@ VG_SOURCES = \
 
 VG_OBJECTS = $(VG_SOURCES:.c=.o)
 
-VG_LIBS = $(GALLIUM_AUXILIARIES) -lm
+VG_LIBS = $(GALLIUM_AUXILIARIES)
+VG_LIB_DEPS = $(EXTRA_LIB_PATH) -lm
 
 ### Include directories
 
@@ -64,7 +65,7 @@ $(TOP)/$(LIB_DIR)/$(VG_LIB_NAME): $(VG_OBJECTS) $(VG_LIBS)
 		-minor $(VG_MINOR) \
 		-patch $(VG_TINY) \
 		-install $(TOP)/$(LIB_DIR) \
-		$(VG_OBJECTS) $(VG_LIBS)
+		$(VG_OBJECTS) $(VG_LIBS) $(VG_LIB_DEPS)
 
 ######################################################################
 # Generic stuff
-- 
1.7.0.3

